src/libs/ActiveClientManager/index.js

```diff
@@ -23,6 +38,16 @@ Onyx.connect({
  */
 function init() {
     Onyx.merge(ONYXKEYS.ACTIVE_CLIENTS, [clientID]);
+    didInitialize = true;
+}
+
+/**
+ * Allow to run any task after ActiveClientManager has successfully initialized
+ *
+ * @returns {Promise<void>}
+ */
+function isReady() {
+    return readyPromise;
```

Comment:
Instead of defining a variable called `readyPromise`, could we just do:

```js
function isReady() {
    return new Promise((res) => {resolveReadyPromise = res;}
}
```

Reply:
Does not serve its original purpose. In case isReady is used at multiple places then all of those promises will not be resolved instead only one would be resolved.